app/assets/stylesheets/_screen.scss

```diff
@@ -361,6 +365,61 @@ a.button {
   margin: 0;
 }
 
+.StoryLink {
+  background: $lightgrey-11;
+  border: 1px solid $lightgrey-13;
+  border-radius: 2px;
+  color: $darkgrey-3;
+  display: inline-block;
+  margin: 1px;
+  padding: 1px 3px;
+
+  &__icon {
+    width: 12px;
+    height: 12px;
+    border-radius: 50%;
+    background: $white;
+    display: inline-block;
+    margin: -2px 0 0 2px;
+    vertical-align: middle;
+
+    &.started  {
+      background: $blue-4;
+    }
+
+    &.finished {
+      background: $butter-1;
+    }
+
+    &.delivered {
+      background: $chameleon-3;
+    }
+
+    &.aux.delivered {
+      margin-left: -1px;
+      background: $darkred;
+    }
+
+    &.mi.accepted,
```

Comment:
It is the class used by the Material Icons library. There are a couple icons using the library, but the other icons are just a span.